packages/enhanced/src/lib/utils/semver.ts

```diff
@@ -0,0 +1,520 @@
+/**
+ * MIT License http://www.opensource.org/licenses/mit-license.php
+ * Author Tobias Koppers @sokra
+ */
+
+'use strict';
+
+import type RuntimeTemplate from 'webpack/lib/RuntimeTemplate';
```

Comment:
Can we replace the semver logic in the runtime with this,

Reply:
Good observation. Let me check

Reply:
@squadronai review